### DISCUSSION TITLE
Count on foreign related table

### DISCUSSION THREAD

**Discussion by BITRU** [supabase/supabase#7886] [Questions]
I would like to perform count on foreign related table. 
Want to find out how many programs each object has. 

```
   const { data: objects, error } = await supabase
          .from('ivs_objects')
          .select('id, name, description, ivs_programs:id("*", { count: "exact" })')
          .eq('company_id', company_id)
```

But In return, I get ivs_programs: foreign id. (not a count). If i remove { count: "exact" } I get array of records as expected. 
Do I have error in syntaxes or this is impossible to perform for foreign table count.

**Comment by steve-chavez** [comment]
Try it like:

```js
   const { data: objects, error } = await supabase
          .from('ivs_objects')
          .select('id, name, description, ivs_programs(count)')
          .eq('company_id', company_id)
```
